packages/core/addon/helpers/get-shallow.ts

```diff
@@ -13,8 +13,8 @@
 import { helper } from '@ember/component/helper';
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-type params = [Dict<any>, string];
-export function getShallow(params: params) {
+type Params = [Dict<any>, string];
+export function getShallow(params: Params) {
```

Comment:
Actually, maybe just do
```suggestion
export function getShallow([obj, key]: [object, string]) {
```

Reply:
Going to keep `Dict<any>` since typescript seems to have problems with using `object` there.
<img width="646" alt="Screen Shot 2020-05-27 at 1 21 41 PM" src="https://user-images.githubusercontent.com/23023478/83057835-1250c380-a01d-11ea-819f-728ef362b858.png">